ci-operator/config/shipwright-io/build/shipwright-io-build-main.yaml

```diff
@@ -0,0 +1,102 @@
+binary_build_commands: make build
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.19
+images:
+- dockerfile_literal: |
+    # Dockerfile to bootstrap build and test in openshift-ci
+    FROM quay.io/openshift/origin-cli:4.14 AS base
```

Comment:
this is not needed, when you want to run oc command at test step, just need to add `cli: 4.14`